packages/graphback-datasync/src/providers/DataSyncConflictProvider.ts

```diff
@@ -0,0 +1,126 @@
+import { ModelDefinition, GraphbackContext, TransformType, FieldTransform, NoDataError, GraphbackOperationType } from '@graphback/core';
+import { DataSyncModelConflictConfig, DataSyncFieldNames, ConflictMetadata, ConflictError } from '../util';
+import { MongoDeltaHelper } from '../deltaHelper';
+import { DataSyncMongoDBDataProvider } from './DatasyncMongoDBDataProvider';
+
+export const MAX_RETRIES = 3;
```

Comment:
```suggestion
export const MAX_RETRIES = 3;
```

It will be useful to be able to configure this. 

@wtrocki what do you think?

It does not have to be done in this PR. We can open an issue to chat about it.

Reply:
Yep.. Not sure why my comments are gone.

Reply:
Opened https://github.com/aerogear/graphback/issues/1856 for this.